_docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/includes/developer-materials-results.html.twig

```diff
@@ -12,77 +12,81 @@
+                    <span class="label material-level-{{ '{{' }} item.fields.level {{ '}}' }}" ng-show="item.fields.level" ng-bind="item.fields.level | firstIfArray"></span>
+                    <div class="material-info">
+                        <h5>
+                            <a ng-href="{{ '{{' }} item.fields.sys_url_view | firstIfArray {{ '}}' }}" ng-bind="item.fields.sys_title | firstIfArray"></a>
+                        </h5>
+                        <span class="contributor" data-sys-contributor="{{ '{{' }} item.fields.sys_author | firstIfArray {{ '}}' }}">
+                            <a class="name"></a>
+                        </span>
+                        <p class="material-datestamp" ng-bind="item.fields.sys_created | timeAgo"></p>
                     </div>
-                    <p>{{ '{{' }} item.fields.sys_description | truncate {{ '}}' }}&hellip;</p></div>
+                    <p ng-bind-template="{{ '{{' }} item.fields.sys_description | truncate {{ '}}' }}&hellip;"></p>
+                </div>
             </li>
         </ul>
     </div>
-    <div class="panel" ng-hide="data.materials.length || data.loading">Sorry there were no results, please modify your
-        search query.
-    </div>
+    <div class="panel" ng-hide="data.materials.length || data.loading" ng-bind-template="Sorry there were no results, please modify your search query."></div>
 </div>
 <div class="large-24" ng-hide="data.loading">
-    <div class="large-7 columns" id="show-entries"><label class="inline">Show</label><select name="show-entries"
-                                                                                             ng-change="fetchAndUpdate();"
-                                                                                             ng-model="pagination.size">
+    <div class="large-7 columns" id="show-entries">
+        <label class="inline">Show</label>
+        <select name="show-entries" ng-change="fetchAndUpdate();" ng-model="pagination.size">
             <option value="10">10</option>
             <option value="15">15</option>
             <option value="20">20</option>
             <option value="25">25</option>
-        </select><label class="inline">entries</label><br/>
-        <p>
-            Showing {{ '{{' }} (paginate.currentPage * pagination.size - pagination.size + 1) | safeNumber {{ '}}' }}
-            to {{ '{{' }} Math.min(data.total, paginate.currentPage * pagination.size) | safeNumber {{ '}}' }}
-            of {{ '{{' }} data.total | safeNumber {{ '}}' }} entries</p></div>
+        </select>
+        <label class="inline">entries</label>
+        <br/>
+        <p ng-bind-template="Showing {{(paginate.currentPage * pagination.size - pagination.size + 1) | safeNumber}} to {{ Math.min(data.total, paginate.currentPage * pagination.size) | safeNumber}} of {{data.total | safeNumber}} entries"></p>
```

Comment:
This needs to be escaped like the others (either {{ '{{' }} ... {{ '}}' }} or {% verbatim %} ... {% endverbatim %} otherwise twig will interpret them.